docs/__common.soy

```diff
@@ -71,6 +71,11 @@ src="https://www.facebook.com/tr?id=1637165926500152&ev=PageView&noscript=1"
   {/if}
   // List buck.css second to override prettify.
   <link type="text/css" rel="stylesheet" href="{ROOT}static/buck.css">
+  <link type="text/css" rel="stylesheet" href="{ROOT}static/search.css">
+  <link
+    type="text/css"
+    rel="stylesheet"
+    href="https://cdn.jsdelivr.net/docsearch.js/1/docsearch.min.css" />
```

Comment:
This was already part of my other pull request. Not sure what happened on the import internally to get it removed :(

https://github.com/facebook/buck/pull/1095